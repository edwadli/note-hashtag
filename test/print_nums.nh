Print (StringOfFloat 42.0)
Print "\n"

Print (StringOfInt 42)
Print "\n"

Print (StringOfFloat (Pow 4.0 2.0))
Print "\n"
