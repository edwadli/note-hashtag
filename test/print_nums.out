42.000000
42
16.000000
